bin/setup

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash 
+set -e
+
+# cd to ~/accounting on login
+if [[ $USER == 'ubuntu' ]]; then
+    mkdir -p ~/accounting
+    cd ~/accounting
+    grep -Fq 'cd ~/accounting' ~/.bashrc || echo 'cd ~/accounting' >> ~/.bashrc
+fi
+
+# Need python-3.6 repository
+echo "Setting up python3.6 repository"
+sudo add-apt-repository 'ppa:deadsnakes/ppa'
+sudo apt-get -y update
+
+# Install all dependencies.
+cd ~/accounting
+echo "Installing dependencies."
+sudo -E apt-get install -y `tr -d '\r' < debian_packages.lst`
```

Comment:
Why doesn't this use the make target? Notice that it's now different, too, so this wouldn't work. Did you actually run this?

Reply:
I did not run it sense merging. After running it I've removed the comments. Why were comments added into the list file?

Reply:
Xavier wanted me to explain in the file why we were downloading `wkhtmltopdf` when it's not actually being used through that package, to not confuse anyone.